set.mm

```diff
@@ -3849,34 +3860,34 @@ the conclusion alone (see ~ pm5.1im and ~ pm5.21im ).  (Contributed by
 
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-  Logical conjunction and disjunction
+  Logical conjunction
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 
-  Here we define disjunction (logical 'or') ` \/ ` ( ~ df-or ) and conjunction
-  (logical 'and') ` /\ ` ( ~ df-an ). We also define various rules for
-  simplifying and applying them, e.g., ~ olc , ~ orc , and ~ orcom .
+  This section defines conjunction of two formulas, denoted by infix `/\` and
+  read "and."  It is defined in terms of implication and negation, which is
```

Comment:
"and".

Reply:
The American grammar puts the period within the quotes, see https://www.grammarly.com/blog/quotation-marks/

Reply:
I also prefer the semantically better `"and".` but unfortunately, American usage puts the period within the quote. As for single/double quote, I think this is byzantine, so I'll let it that way, but you can change them later.

A quick search through set.mm shows that neither of these rules is applied consistently.

Reply:
One could actually conceive a case where it is misleading:

The ASCII translation for the operator "for all" is "A."

Reply:
Well, '."' is used many times in set.mm, so it seems to be OK.

Reply:
Just in case this particular thread of discussion blocks the merge, the result is good enough for me.  I don't oppose to merging it.

Reply:
There's more than anyone probably wants to read about this topic at https://en.wikipedia.org/wiki/Wikipedia:Logical_quotation_on_Wikipedia . Personally, I'd vote for the logical quotation style but I'm not sure how deep we need to get into trying to standardize this sort of thing.

Reply:
@jkingdon: it would certainly be nice to standardize to the logical quotation style. Not volunteering for a global update though... I'm not sure how hard it would be (e.g., does a search for `".` and `",` yield some false positives?) I'll use logical style from now on.

Reply:
I would also prefer the "logical quotation style" (especially because metamath is about logic ;-) ). Then the sentence in the comment discussed above would be changed to:

> This section defines conjunction of two formulas, denoted by infix `/\` and read "and".

I would propose to add this to our conventions, adding the remark that this style should be used from now on, and existing violations should be corrected on occasion. Or is someone volunteering for a global update? @david-a-wheeler @wlammen would you agree?

Reply:
I also prefer the logical quotation style over typographical quotation style. In technical works the typographical quotation style often leads to misleading statements and errors, so I think many mathematical and computer science books are using logical style. Wikipedia requires it. [The Jargon File](http://www.catb.org/~esr/jargon/html/writing-style.html) talks about it. They're sometimes called "British" vs. "American" style, but that's a little misleading; publications on both sides of the pond use both styles. See: https://en.wikipedia.org/wiki/Wikipedia:Logical_quotation_on_Wikipedia

I suggest adding it to the set of conventions for *future* changes. We can then slowly transform as we go, and commit a mass-change of the rest at some time in the future.

Reply:
I created an issue that proposes using logical quotation here: https://github.com/metamath/set.mm/issues/2786